src/services/notification/slack.rs

```diff
@@ -151,7 +142,7 @@ mod tests {
 		variables.insert("status".to_string(), "critical".to_string());
 
 		let result = notifier.format_message(&variables);
-		assert_eq!(result, "*Alert*\n\nValue is 100 and status is critical");
+		assert_eq!(result, "Value is 100 and status is critical");
```

Comment:
Same here, let's not modify the tests. Let's fix the implementation. Tests are our source of truth.